src/pages/SignIn/SignIn.tsx

```diff
@@ -0,0 +1,59 @@
+import { Link, useNavigate } from 'react-router-dom';
+import Button from '../../components/Button/Button';
+import Form from '../../components/Form/Form';
+import FormInputs from '../../components/FormInputs/FormInputs';
+import Input from '../../components/Input/Input';
+import Popup from '../../components/Popups/Popup/Popup';
+import styles from './SignIn.module.scss';
+import { FieldValues, SubmitHandler, useForm } from 'react-hook-form';
+import { regexPassword } from '../../utils/consts';
+import InputPhone from '../../components/InputPhone/InputPhone';
+import { authApi } from '../../utils/api/auth';
+import { useTranslation } from 'react-i18next';
+import { useState } from 'react';
+
+const SignIn = () => {
+    const [showError, setShowError] = useState<boolean>(false);
+    const [errorMessage, setErrorMessage] = useState<string | undefined>('')
+    const navigate = useNavigate();
+    const { t } = useTranslation();
+    const {
+        register,
+        handleSubmit,
+        formState: { errors },
+    } = useForm();
+    const onSubmit: SubmitHandler<FieldValues> = async (data) => {
+        const { password, phoneNumber } = data;
+        const res = await authApi.login({ phone: phoneNumber, password });
```

Comment:
```suggestion
        setErrorMessage(null); 
        const res = await authApi.login({ phone: phoneNumber, password });
```
чтобы юзер понимал, что что-то происходит
А еще по-хорошему нужно как-то показывать, что форма загружается. это следующим шагом сделаем https://github.com/orgs/bronfood-com/projects/1/views/1?pane=issue&itemId=48512877

Reply:
Добавил сброс ошибки